src/e2e-test/features/salesforcebatchsource/DesignTime.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Salesforce Batch Source - Design time scenarios
+
+  Scenario: Verify user should be able to get output schema for a valid SOQL query
+    When I Open CDF Application
+    And I select data pipeline type as Data Pipeline - Batch
+    And I select data pipeline source as Salesforce
+    And I configure Salesforce source for a basic SOQL query "SELECT Id, Name, Phone FROM Account"
```

Comment:
Actual query should only be read from the properties file. Use reference name here.